modules/data-entry/src/main/frontend/src/questionnaire/Subject.jsx

```diff
@@ -282,7 +282,7 @@ function SubjectMember (props) {
                           {entry["statusFlags"].map((status) => {
                             return <Chip
                               label={wordToTitleCase(status)}
-                              className={`${classes.subjectChip} ${"INVALID" === status ? classes.errorChip : classes.warningChip}`}
+                              className={`${classes.subjectChip} ${classes[status + "Chip"] ? classes[status + "Chip"] : classes.DefaultChip}`}
```

Comment:
This should do the same thing

```suggestion
                              className={`${classes.subjectChip} ${classes[status + "Chip"] || classes.DefaultChip}`}
```